### ISSUE TITLE
Dota Application closes when Rubick uses Telekinesis: Match ID 6773120530

### ISSUE THREAD

**Issue by edcelf007** [ValveSoftware/Dota2-Gameplay#3349]
### Description

Dota Application closes everytime when Rubick uses Telekinesis starting at in-game time stamp 35:57 onwards. in-game time stamp 38:19, 40:58, 48:20.

### Example Match ID (and possibly Timestamp)

6773120530 - 35:57, 38:19, 40:58, 48:20.

**Comment by maitzeth** [comment]
Same here, plz take a look.

**Comment by jpaolopapa** [comment]
Yeah this happened to me just now. The issue is when you buy a shard.

**Comment by edcelf007** [comment]
Also tried on Demo Hero using rubick same result. Telekinesis Shard causes the Dota Application to close suddenly

**Comment by memoAramburu** [comment]
hello, it happens to me when I use the ability Telekinesis Land.

Match ID 6773152177

**Comment by dron1t** [comment]
duplicate of: #3346
